src/site_upload/powerset_merge/powerset_merge.py

```diff
@@ -118,56 +118,67 @@ def merge_powersets(manager: s3_manager.S3Manager) -> None:
     for latest_path in latest_file_list:
         if manager.version not in latest_path:
             continue
-        site_specific_name = functions.get_s3_site_filename_suffix(latest_path)
+        latest_metadata = functions.parse_s3_key(latest_path)
         subbucket_path = (
-            f"{manager.study}/{manager.study}__{manager.data_package}" f"/{site_specific_name}"
-        )
-        date_str = datetime.datetime.now(datetime.UTC).isoformat()
-        timestamped_name = f".{date_str}.".join(site_specific_name.split("."))
-        timestamped_path = (
-            f"{manager.study}/{manager.study}__{manager.data_package}" f"/{timestamped_name}"
+            f"{manager.study}/{manager.study}__{manager.data_package}/{latest_metadata.site}/" 
+            f"{manager.study}__{manager.data_package}__{manager.version}"
         )
+        archives = []
```

Comment:
nit: maybe something that is more descriptive - like `archived_filed` or `moved_files` or something? To indicate that this is a record of what we did, so we can undo it.